apps/main/src/llamalend/features/bands-chart/BandsChart.tsx

```diff
@@ -0,0 +1,188 @@
+    if (!instance) return
+    instance.resize()
+  }, [height, finalOption])
+
+  if (!chartData?.length) {
+    return (
+      <Box sx={{ width: '100%', fontVariantNumeric: 'tabular-nums', height }}>
+        <Box
+          sx={{
+            display: 'flex',
+            alignItems: 'center',
+            justifyContent: 'center',
+            height: '100%',
+            fontSize: '14px',
+            color: palette.textColor,
+          }}
+        >
+          {isLoading ? (
+            // TODO: update when re-theming the candle chart to match spinners
+            <SpinnerWrapper>
+              <Spinner size={18} />
+            </SpinnerWrapper>
+          ) : (
+            <EmptyState isError={isError} />
+          )}
+        </Box>
+      </Box>
+    )
+  }
+
+  const rawValues: [number, number] = [
+    typeof zoomRange.startValue === 'number' ? zoomRange.startValue : priceMin,
+    typeof zoomRange.endValue === 'number' ? zoomRange.endValue : priceMax,
+  ]
+  const normalizedValues: [number, number] =
+    rawValues[0] <= rawValues[1] ? [rawValues[0], rawValues[1]] : [rawValues[1], rawValues[0]]
+
+  return (
+    <Stack
+      direction="row"
+      sx={{ width: '100%', minHeight: `${height}px`, height, gap: 2, minWidth: 0, alignItems: 'stretch' }}
+    >
+      <Box
+        ref={containerRef}
+        sx={{
+          display: 'flex',
+          fontVariantNumeric: 'tabular-nums',
+          flex: 1,
+          minWidth: 0,
+          height: '100%',
+          '& *': { cursor: 'default !important' },
+        }}
+      >
+        <ReactECharts
+          ref={chartRef}
+          option={finalOption}
+          style={{ width: '100%', height: '100%' }}
+          opts={{ renderer: 'canvas' }}
+          onEvents={{ datazoom: onDataZoom }}
+          notMerge={true}
+          lazyUpdate={true}
+          autoResize={false}
+        />
+      </Box>
+      <Slider
+        orientation="vertical"
+        size="small"
+        value={normalizedValues}
+        min={priceMin}
+        max={priceMax}
+        onChange={(_e, val) => {
+          if (Array.isArray(val) && val.length === 2 && typeof val[0] === 'number' && typeof val[1] === 'number') {
+            const [minV, maxV] = val[0] <= val[1] ? [val[0], val[1]] : [val[1], val[0]]
+            setZoomRange({ startValue: minV, endValue: maxV })
+          }
+        }}
+        onChangeCommitted={(_e, val) => {
+          if (Array.isArray(val) && val.length === 2 && typeof val[0] === 'number' && typeof val[1] === 'number') {
+            const [minV, maxV] = val[0] <= val[1] ? [val[0], val[1]] : [val[1], val[0]]
+            setZoomRange({ startValue: minV, endValue: maxV })
+          }
+        }}
+        data-rail-background="filled"
+        sx={{
+          // disable MUI's default padding for coarse touch devices to match design
+          '@media (pointer: coarse)': {
+            padding: `0 !important`,
```

Comment:
Can't we use `&` to increase importance instead of `!important`? I expect a comment whenever using `!important`